src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -479,6 +479,7 @@ struct SvgStyleStroke
 
 struct SvgStyleProperty
 {
+    bool display;
```

Comment:
Please keep this at the end of the structure for packing the data. Probably, you can align the boolean values together.
```
{
   ...
   bool curColorSet
   bool paintOrder;
   bool display;
}

Reply:
done

Reply:
I expect we could save padding memory 1 ~ 5 bytes if you put them at the end of the structure.

Reply:
ofc, sorry, done